easybuild/easyconfigs/h/HDF5/HDF5-1.8.18-foss-2016b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'HDF5'
+version = '1.8.18'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/current18/src/']
```

Comment:
@jhein32 My suggestion is to do this, so the download of 1.8.18 doesn't break in the future when they move it to the location where the others are:

```python
source_urls = [
    'https://support.hdfgroup.org/ftp/HDF5/current18/src/',
    'https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(versions)/src/',
]
```

How sure are you that the `18` in `current18` come from the last part of `1.8.18`, and is not equivalent with `%(version_major)s%(version_minor)s` ? The latter makes more sense to me...